proton-c/bindings/python/proton/reactor.py

```diff
@@ -565,7 +565,7 @@ def _connect(self, connection, reactor):
             if not self.ssl_domain:
                 raise SSLUnavailable("amqps: SSL libraries not found")
             self.ssl = SSL(transport, self.ssl_domain)
-            self.ssl.peer_hostname = url.host
+            self.ssl.peer_hostname = self.virtual_host if self.virtual_host != None else url.host
```

Comment:
I'm pretty sure that None has truthiness of false so this can be written more simply as:
... = self.virtual_host if self.virtual_host else url.host